flink-end-to-end-tests/test-scripts/test_ha_per_job_cluster_datastream.sh

```diff
@@ -153,6 +153,6 @@ function run_ha_test() {
 STATE_BACKEND_TYPE=${1:-file}
 STATE_BACKEND_FILE_ASYNC=${2:-true}
 STATE_BACKEND_ROCKS_INCREMENTAL=${3:-false}
-ZOOKEEPER_VERSION=${4:-3.4}
+ZOOKEEPER_VERSION=${4:-3.5}
```

Comment:
Like in the previous comment: Shall we move this into a variable since it's used in two places? 🤔